utils/index.d.ts

```diff
@@ -1538,17 +1540,10 @@ export declare abstract class AzureWizardPromptStep<T extends IActionContext> {
     public supportsDuplicateSteps: boolean;
 
     /**
-     * Optional id used to determine if this step is unique, for things like caching values and telemetry
+     * A id used to determine if this step is unique, for things like caching values and telemetry
      * If not specified, the class name will be used instead
```

Comment:
Actually might want to update this comment by deleting this line - it says if the id is not specified the class name will be used instead which isn't true anymore